src/librustc_trans/trans/type_of.rs

```diff
@@ -405,7 +417,14 @@ pub fn in_memory_type_of<'a, 'tcx>(cx: &CrateContext<'a, 'tcx>, t: Ty<'tcx>) ->
       }
       ty::TyStruct(did, ref substs) => {
           if t.is_simd(cx.tcx()) {
-              let llet = in_memory_type_of(cx, t.simd_type(cx.tcx()));
+              let e = t.simd_type(cx.tcx());
+              println!("in_memory_type_of: simd: {}", e);
```

Comment:
This seems to be a stray debugging println!, which is hit while compiling libcore.